pypeit/core/telluric.py

```diff
@@ -135,22 +134,73 @@ def pca_eval(theta,pca_dict):
 #        theta_pca[2:npca+1]``
 #
 #    """
-#    gaussian_mixture_model = pca_dict['prior']
+#    gaussian_mixture_model = qso_pca_dict['prior']
 #    A = theta[2:]
 #    return gaussian_mixture_model.score_samples(A.reshape(1,-1))
 
 
+def read_telluric_pca(filename, wave_min=None, wave_max=None, pad_frac=0.10):
+    """
+    Reads in the telluric PCA components from a file.
+
+    Optionally, this method also trims wavelength to be in within ``wave_min``
+    and ``wave_max`` and pads the data (see ``pad_frac``).
+
+    Args:
+        filename (:obj:`str`):
+            Telluric PCA filename
+        wave_min (:obj:`float`):
```

Comment:
Add optional: `wave_min (:obj:'float', optional)`, and likewise for the other optional arguments

Reply:
Done.